src/crime_vancouver_eda.py

```diff
@@ -0,0 +1,91 @@
+# author: Group 24
+# date: 2021-11-25
+
+''' Creates eda plots for the pre-processed training data from the Crime Vancouver Data 
+    (from https://geodash.vpd.ca/opendata/) and saves the plots as png files
+
+Usage: src/eda_crime.py --input_path=<input_path> --out_dir=<out_dir>
+  
+Options:
+--input_path=<train_path>     Path (including filename) to training data (which needs to be saved as a .csv file)
+--out_dir=<out_dir>           Path to directory where the plots should be saved
+'''
+
+import altair as alt
+import pandas as pd
+import numpy as np
+import dataframe_image as dfi
+from sklearn.model_selection import train_test_split
+alt.data_transformers.enable('data_server')
+alt.renderers.enable('mimetype')
+from docopt import docopt
+
+opt = docopt(__doc__)
+
+def main(input_path, out_dir):
+    
+    # Read data from csv and get the train data
+    df = pd.read_csv(input_path)
+    df = df.query('2010 < YEAR <= 2020')
+    train_df, test_df = train_test_split(df, test_size=0.20, random_state=123)
```

Comment:
I think the input_path should be data/processed/training_feature.csv. So you should not split it here.
Again, if your report want to make it 75/25, put it as an issue and I will modify split_data.py.
Please update the markdown description accordingly if needed.

Reply:
I agree with Thomas take as input the files inside /data/processed/

Reply:
Cool, will delete line 30 here. I believe I should keep line 20 still?

Reply:
Do you mean line 29 instead of 30? As during the split_data.py I should have filtered the year range out.
In this case, remove both 29 and 30 and I will update the split_data.py to cover only 2010 to 2020 as our master copy of data.

I've created an issue for awareness #31

Reply:
Great, this is solved now